core/serializers.py

```diff
@@ -0,0 +1,49 @@
+from collections.abc import Sequence
+from typing import ClassVar
+
+from django.db.models import Model
+from rest_framework import serializers
+
+from core.models import Community, Post
+from users.models import User
+
+
+class PostSerializer(serializers.ModelSerializer):
+    score = serializers.ReadOnlyField()
+
+    class Meta:
+        model: Model = Post
+        exclude: ClassVar[Sequence[str] | str] = ["version"]
```

Comment:
Do not use exclude please, use fields, and always define which fields. 
Reason: we want to see exactly what is passed. This sends all fields except version. Now if we add some private field to the model it wil automatically show up here. 
Let's say document id, or whatever. And it shows in api. So it's a private data leak. 
As a convention, do not use exclude and do not use __all__. Always define exact fields to show in API.